packages/sui-studio/src/components/tryRequire.js

```diff
@@ -1,30 +1,7 @@
 /* global __BASE_DIR__ */
-const requireContextSrc = require.context(
```

Comment:
Stop using problematic require.context of import files and rely on dynamic import when possible.